docs/resources/resource_policy.md

```diff
@@ -0,0 +1,145 @@
+        )
+                when {
+                context.project.ipAccessList.contains(ip("0.0.0.0/0"))
+        };
+EOF
+    },
+  ]
+}
+
+resource "mongodbatlas_resource_policy" "cloud_provider" {
+  org_id = var.org_id
+  name   = "forbid-cloud-provider"
+  policies = [
+    {
+      body = templatefile("${path.module}/cloud-provider.cedar", {
+        CLOUD_PROVIDER = "azure"
+      })
+    },
+    {
+      body = templatefile("${path.module}/cloud-provider.cedar", {
+        CLOUD_PROVIDER = "aws"
+      })
+    },
+  ]
+}
+
+data "cedar_policyset" "cloud_region" {
+  policy {
+    any_principal = true
+    effect        = "forbid"
+    action = {
+      type = " cloud::Action"
+      id   = "cluster.createEdit"
+    }
+    any_resource = true
+    when {
+      text = "context.cluster.regions.contains(cloud::region::\"gcp:us-east1\")"
+    }
+  }
+}
+
+resource "mongodbatlas_resource_policy" "cloud_region" {
+  org_id = var.org_id
+  name   = "forbid-cloud-region"
+  policies = [
+    {
+      body = data.cedar_policyset.cloud_region.text
+    },
+  ]
+}
+
+
+data "mongodbatlas_resource_policy" "project_ip_access_list" {
+  org_id = mongodbatlas_resource_policy.project_ip_access_list.org_id
+  id     = mongodbatlas_resource_policy.project_ip_access_list.id
+}
+
+data "mongodbatlas_resource_policies" "this" {
+  org_id = data.mongodbatlas_resource_policy.project_ip_access_list.org_id
+
+  depends_on = [mongodbatlas_resource_policy.project_ip_access_list, mongodbatlas_resource_policy.cloud_provider, mongodbatlas_resource_policy.cloud_region]
+}
+
+
+output "policy_ids" {
+  value = { for policy in data.mongodbatlas_resource_policies.this.resource_policies : policy.name => policy.id }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) Human-readable label that describes the atlas resource policy.
+- `org_id` (String) Unique 24-hexadecimal digit string that identifies the organization that contains your projects. Use the [/orgs](#tag/Organizations/operation/listOrganizations) endpoint to retrieve all organizations to which the authenticated user has access.
+- `policies` (Attributes List) List of policies that make up the atlas resource policy. (see [below for nested schema](#nestedatt--policies))
+
+### Read-Only
+
+- `created_by_user` (Attributes) The user that last updated the atlas resource policy. (see [below for nested schema](#nestedatt--created_by_user))
+- `created_date` (String) Date and time in UTC when the atlas resource policy was created.
+- `id` (String) Unique 24-hexadecimal digit string that identifies an atlas resource policy.
+- `last_updated_by_user` (Attributes) The user that last updated the atlas resource policy. (see [below for nested schema](#nestedatt--last_updated_by_user))
+- `last_updated_date` (String) Date and time in UTC when the atlas resource policy was last updated.
+- `version` (String) A string that identifies the version of the atlas resource policy.
```

Comment:
```suggestion
- `created_by_user` (Attributes) The user that last updated the Atlas resource policy. (see [below for nested schema](#nestedatt--created_by_user))
- `created_date` (String) Date and time in UTC when the Atlas resource policy was created.
- `id` (String) Unique 24-hexadecimal digit string that identifies an Atlas resource policy.
- `last_updated_by_user` (Attributes) The user that last updated the Atlas resource policy. (see [below for nested schema](#nestedatt--last_updated_by_user))
- `last_updated_date` (String) Date and time in UTC when the Atlas resource policy was last updated.
- `version` (String) A string that identifies the version of the Atlas resource policy.
```

Reply:
Changed in a8851acdb8c44933b67c874a3160f5b6938a5ba2